### PR TITLE
Snack bar/consecutive snack bars

### PR DESCRIPTION
**Pull request by ZafarSaeedKhan**

### CODE REVIEW

File: src/lib/components/snackbar/mdl-snackbar.service.ts [hunk 1]
```diff
@@ -88,9 +88,9 @@ export interface IMdlSnackbarMessage {
 export class MdlSnackbarService {
 
   private cFactory: ComponentFactory<any>;
+  previousSnack: {component: MdlSnackbarComponent, cRef: ComponentRef<any>};
```
Comment (mseemann) [inline src/lib/components/snackbar/mdl-snackbar.service.ts]: please make it explicitly private

File: src/lib/components/snackbar/mdl-snackbar.service.ts [hunk 2]
```diff
@@ -108,24 +108,42 @@ export class MdlSnackbarService {
...
-    // observable? push the configured instance and consume one after another?
+    this.previousSnack = {
+      component: mdlSnackbarComponent,
+      cRef: cRef
+    };
 
     if (snackbarMessage.action) {
+      if(closeAfterTimeout) {
+        setTimeout( () => {
+          mdlSnackbarComponent.hide()
```
Comment (mseemann) [inline src/lib/components/snackbar/mdl-snackbar.service.ts]: this seem to be a code duplication - please extract a function

**Comment by mseemann** [comment]
except two comments. everything else looks good to me 👍  - i have tested it locally...

File: src/lib/components/snackbar/mdl-snackbar.service.ts (hunk 2 above)
Comment (ZafarSaeedKhan) [inline src/lib/components/snackbar/mdl-snackbar.service.ts]: Done

File: src/lib/components/snackbar/mdl-snackbar.service.ts (hunk 1 above)
Comment (ZafarSaeedKhan) [inline src/lib/components/snackbar/mdl-snackbar.service.ts]: Done

**Comment by ZafarSaeedKhan** [comment]
Refactored with the instructed changes :)

**Comment by mseemann** [comment]
great! 👍  will be in the next release!

**Comment by ZafarSaeedKhan** [comment]
When will you roll out the latest release?

**Comment by mseemann** [comment]
next week on monday. do you need this functionality asap?

**Pull request merged**